From c6ca107d17ca1bc023dc455c1f733055572b8ed9 Mon Sep 17 00:00:00 2001
From: Yoni Izutskiver <yizutskiver@habana.ai>
Date: Sun, 09 Jan 2022 20:24:43 +0200
Subject: [PATCH] [SW-69272] gaudi2: pci: fix msix pba offset

The HW reset value for MSIx PBA offset register is incorrect, and should
be overwritten before the PCIe link is enabled. Originally the overwrite
was done with a wrong value. Change the write value to 0x2002.

Change-Id: I3720eb3ba306bd48b177eb535df5684a630d63e4
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index e99fca7..1e858fc 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -704,7 +704,7 @@
 	}
 }
 
-#define MSIX_PBA_OFFSET_REG_VAL		0x400
+#define MSIX_PBA_OFFSET_REG_VAL	0x2002
 
 void set_pci_msix_pba_offset_val(void)
 {
